src/main/java/no/unit/nva/cristin/projects/PresentationConverter.java

```diff
@@ -42,8 +43,14 @@ protected ProjectPresentation asProjectPresentation(Project project, String lang
             fundingSourcePresentation.fundingSourceCode = fundingSource.fundingSourceCode;
             fundingSourcePresentation.projectCode = fundingSource.projectCode;
             fundingSourcePresentation.names = fundingSource.fundingSourceName.entrySet().stream()
-                    .map(name -> new FundingSourceNamePresentation(name.getKey(), name.getValue()))
-                    .collect(Collectors.toList());
+                .map(name -> {
+                    FundingSourceNamePresentation fundingSourceNamePresentation =
```

Comment:
extract method

Reply:
This class is old (related to the old data model) and will most probably be deleted in next PR